src/schedule/schedule-week-view/slot-content.tsx

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import { TimeHelper } from "../../util/time-helper";
+import { PositionedSlot } from "./types";
+import {
+    SlotContent,
+    SlotServiceName,
+    SlotAvailability,
+} from "./schedule-week-view.styles";
+
+interface SlotContentComponentProps {
+    positionedSlot: PositionedSlot;
+    blockedMessage?: string;
+}
+
+export const SlotContentComponent: React.FC<SlotContentComponentProps> = ({
```

Comment:
maybe can rename this to `SlotContent`, and the container can be `SlotContentContainer`